### PR TITLE
Update examples to give clear error messages

### PR DESCRIPTION
**Pull request by aakashapoorv**
This PR updates the example scripts to  confirms that the `max_seq_len` is within an acceptable range (`1 <= max_seq_len <= 8192`), and the given `ckpt_dir` (checkpoint directory),  `tokenizer_path` (tokenizer file paths) exist and gives clear error messages.

### CODE REVIEW

**Comment by subramen** [comment]
Thanks @aakashapoorv I think it might be better to add the asserts in the `build` function instead of in the example scripts. https://github.com/meta-llama/llama3/blob/cc44ca2e1c269f0e56e6926d7f4837c983c060dc/llama/generation.py#L37

**Comment by aakashapoorv** [comment]
@subramen Thanks for the suggestion! Thats better, I've moved the assertions into the build function of the Llama class 😊

File: example_text_completion.py [hunk 1]
```diff
@@ -61,4 +61,4 @@ def main(
 
 
 if __name__ == "__main__":
-    fire.Fire(main)
+    fire.Fire(main)
```
Comment (subramen) [inline example_text_completion.py:64]: what is the change here (and in the other example script)?

Comment (aakashapoorv) [reply example_text_completion.py]: Fixed the `No newline at end of file` issue in this and the other example script by enabling the VS Code setting to automatically add a newline 😅

**Review by subramen — APPROVED** [review]

**Pull request merged**